README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio: setup and mobile version skeleton
+# My Portfolio
```

Comment:
- [ ] Nice deploying your project to GitHub pages, I had to figure out the link @ (https://pirc25.github.io/Portfolio/), kindly add the link to your Readme File so people can easily access it.